app/models/govuk_publishing_components/audit_components.rb

```diff
@@ -38,18 +48,16 @@ def initialize(path)
       }
     end
 
-  private
-
     def find_files(files, replace)
       files.map { |file| clean_file_name(file.gsub(replace, "")) }.sort
     end
 
     def clean_file_name(name)
-      name.tr("/_-", " ").gsub(".html.erb", "").gsub(".scss", "").gsub(".js", "").gsub("spec", "").gsub(".rb", "").strip
+      name.tr("/_-", " ").gsub(".html.erb", "").gsub(".erb", "").gsub(".scss", "").gsub(".js", "").gsub("spec", "").gsub(".rb", "").strip
```

Comment:
Earlier there was a change from `*.html.erb` to `*erb` - does this change need to happen here as well?
```suggestion
      name.tr("/_-", " ").gsub(".erb", "").gsub(".erb", "").gsub(".scss", "").gsub(".js", "").gsub("spec", "").gsub(".rb", "").strip
```

Reply:
Good spot! Actually no. We want to remove the file extension from the filename, but it might have found an `.erb` file or `.html.erb` file, because we're looking for `.erb`, which could be either.